NamingSystems/ods-organization-code-namingsystem.xml

```diff
@@ -0,0 +1,36 @@
+<NamingSystem xmlns="http://hl7.org/fhir">
+  <id value="ods-organization-code-namingsystem" />
+  <name value="ODSOrganisationCode" />
+  <status value="draft" />
+  <kind value="identifier" />
+  <date value="2023-08-01" />
+  <publisher value="NHS England" />
+  <contact>
+        <name value="Organisation Data Service" />
+        <telecom>
+            <system value="email" />
+            <value value="exeter.helpdesk@nhs.net" />
+            <use value="work" />
+            <rank value="1" />
+        </telecom>
+    </contact>
+  <responsible value="Organisation Data Service" />
+  <description value="The identifier system for an Organisation registered with the Organisation Data Service (ODS)." />
```

Comment:
I think Organisation should either be 'organisation' (small o and s) or 'Organization' (big o and little z for FHIR asset). Same as line 19 too.

Reply:
Good cactch, this will need doing in UK Core